src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2.java

```diff
@@ -224,4 +222,19 @@ public void closeTool() {
             m2Engine.shutdown();
         }
     }
+
+    private Set<VCFHeaderLine> getMutect2VCFHeaderLines() {
```

Comment:
I'm on board with this refactor. I think it's worth revisiting after the subclassing I suggested above.